ls1intum/Artemis
# `Development`: Add documentation for lectures

### Checklist
#### General



### Motivation and Context



Add page about `Lectures` to documentation.

### Description


https://artemis-platform--5373.org.readthedocs.build/en/5373/user/lectures/

### Steps for Testing

- Checkout the new documentation page (link above)
- Ensure that the link to the documentation page works correctly on the About Us page and in the Readme

### Review Progress





#### Code Review
